impl/src/test/resources/ClientProxyProviderTest/META-INF/services/org.jboss.weld.bean.proxy.ClientProxyProvider

```diff
@@ -0,0 +1 @@
+org.jboss.weld.tests.unit.proxy.ClientProxyProviderTest$FancyClientProxyProvider
```

Comment:
Isn't this going to interfere with other tests? As in, won't all other unit tests here have this proxy provider now?
Might be better to put this in some of the tests using arquillian as you can control whats on CP and also declare this asset programmatically for just that one test.

Reply:
I don’t think so. My intent was for this to apply only to the current unit test.  What did I miss?